src/main/java/fr/insee/genesis/controller/rest/responses/ResponseController.java

```diff
@@ -62,15 +65,26 @@ public class ResponseController {
 
     private static final String DDI_REGEX = "ddi[\\w,\\s-]+\\.xml";
     public static final String S_S = "%s/%s";
+    private static final String CAMPAIGN_ERROR = "Error for campaign {} : {}";
```

Comment:
Typo : en anglais il n'y a pas d'espace avant les ":" 

`"Error for campaign {}: {}"`